x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -674,7 +672,6 @@ public void authenticationFailed(String requestId, String realm, AuthenticationT
                 .with(PRINCIPAL_FIELD_NAME, token.principal())
                 .withRestUriAndMethod(request)
                 .withRestOrigin(threadContext)
-                .withRequestBody(request)
```

Comment:
Authn failed & similar events are going to not display the request body anymore.